src/Goerli-DssSpell.sol

```diff
@@ -20,72 +20,79 @@ pragma experimental ABIEncoderV2;
-            join:                  MCD_JOIN_WSTETH_A,
-            clip:                  MCD_CLIP_WSTETH_A,
-            calc:                  MCD_CLIP_CALC_WSTETH_A,
-            pip:                   PIP_WSTETH,
-            isLiquidatable:        true,
-            isOSM:                 true,
-            whitelistOSM:          true,
-            ilkDebtCeiling:        3 * MILLION,
-            minVaultAmount:        10000,
-            maxLiquidationAmount:  3 * MILLION,
-            liquidationPenalty:    1300,        // 13% penalty fee
-            ilkStabilityFee:       FOUR_PCT_RATE,
-            startingPriceFactor:   13000,       // Auction price begins at 130% of oracle
-            breakerTolerance:      5000,        // Allows for a 50% hourly price drop before disabling liquidations
-            auctionDuration:       140 minutes,
-            permittedDrop:         4000,        // 40% price drop before reset
-            liquidationRatio:      16000,       // 160% collateralization
-            kprFlatReward:         300,         // 300 Dai
-            kprPctReward:          10           // 0.1%
-        }));
-        DssExecLib.setStairstepExponentialDecrease(MCD_CLIP_CALC_WSTETH_A, 90 seconds, 9900);
-        DssExecLib.setIlkAutoLineParameters("WSTETH-A", 50 * MILLION, 3 * MILLION, 8 hours);
+    constructor() public {
+        WBTC = DssExecLib.getChangelogAddress("WBTC");
+        PIP_WBTC = DssExecLib.getChangelogAddress("PIP_WBTC");
+    }
 
-        DssExecLib.setChangelogAddress("STETH", STETH);
-        DssExecLib.setChangelogAddress("WSTETH", WSTETH);
-        DssExecLib.setChangelogAddress("PIP_WSTETH", PIP_WSTETH);
-        DssExecLib.setChangelogAddress("MCD_JOIN_WSTETH_A", MCD_JOIN_WSTETH_A);
-        DssExecLib.setChangelogAddress("MCD_CLIP_WSTETH_A", MCD_CLIP_WSTETH_A);
-        DssExecLib.setChangelogAddress("MCD_CLIP_CALC_WSTETH_A", MCD_CLIP_CALC_WSTETH_A);
+    function actions() public override {
 
-        // Rely Median to Oracles team
+        //  Add WBTC-B as a new Vault Type
+        //  https://vote.makerdao.com/polling/QmSL1kDq?network=mainnet#poll-detail (WBTC-B Onboarding)
+        //  https://vote.makerdao.com/polling/QmRUgsvi?network=mainnet#poll-detail (Stability Fee)
+        //  https://forum.makerdao.com/t/wbtc-b-collateral-onboarding-risk-assessment/11397
+        //  https://forum.makerdao.com/t/signal-request-new-iam-vault-type-for-wbtc-with-lower-lr/5736
+        DssExecLib.addNewCollateral(
+            CollateralOpts({
+                ilk:                   "WBTC-B",
+                gem:                   WBTC,
+                join:                  MCD_JOIN_WBTC_B,
+                clip:                  MCD_CLIP_WBTC_B,
+                calc:                  MCD_CLIP_CALC_WBTC_B,
+                pip:                   PIP_WBTC,
+                isLiquidatable:        true,
+                isOSM:                 true,
+                whitelistOSM:          false,
```

Comment:
It is fine to leave this one in false, as this is already set. But honestly it might be confusing for the future as a new collateral with a first used token with a median + osm needs this on `true`.
The flag itself is thought to put in `false` when the `osm` source is a `ds-value` instead of a median, which actually never happened.

Reply:
I think we might want to revisit a bit these flags but for now I'd just put in `true`.
cc @brianmcmichael

Reply:
Done in e0d4f5f.